configs.ts

```diff
@@ -0,0 +1,55 @@
+// This will import all variables from any `.env` in your root directory and make them available to the deno process via `Deno.env.get("MY_VAR")`
+import "https://deno.land/x/dotenv/load.ts";
+
+// Step 1: Add all your bot's information below. The only required one is token and prefix.
```

Comment:
This is a good oppurtunity to reshape things here.

```ts
// Step 1. REQUIRED. This line will add make your bot token available.
export const token = Deno.env.get("TOKEN") || "";
```

Reply:
Is it?
I'm kind of losing steam overhear. ^^

I guess it's easy enough for someone else to change after this is merged.